bin/docker/raphtory/DockerfileV2

```diff
@@ -0,0 +1,38 @@
+ARG VERSION
+FROM hseeberger/scala-sbt:eclipse-temurin-11.0.14.1_1.6.2_2.13.8 as sbt-builder
+COPY core core/
+COPY examples examples/
+COPY project project/
+COPY deploy deploy/
+COPY build.sbt build.sbt
+COPY .scalafmt.conf .scalafmt.conf
+RUN sbt clean core/assembly
+
+FROM eclipse-temurin:11.0.15_10-jdk as python-build
```

Comment:
should be an arg for version for easy changes/testing of new versions

Reply:
Yes, I had some trouble setting that up